### DISCUSSION TITLE
Check which Github Api calls made by an account

### DISCUSSION THREAD

**Discussion by tungjb** [community/community#24577] [Apps, API and Webhooks]
<p>One of our service account hit Git Api rate limit of 5000 per hour. It is used in jenkins and Aws code pipeline.</p>
<p>Is there anyway that to figure out which Github API calls was made by this account? So that we can investigate which plugins caused the api rate limit. In Jenkins logs, the rate decreases quite quickly:</p>
<p><em>Mar 31, 2020 5:38:24 PM FINE org.kohsuke.github.GitHub</em><br>
<em>Rate limit now: GHRateLimit{remaining=4242, limit=5000, resetDate=Tue Mar 31 18:31:19 AEDT 2020}</em><br>
<em>Mar 31, 2020 5:59:17 PM FINE org.kohsuke.github.GitHub</em><br>
<em>Rate limit now: GHRateLimit{remaining=2044, limit=5000, resetDate=Tue Mar 31 18:31:19 AEDT 2020}</em><br>
<em>Mar 31, 2020 5:59:17 PM FINE org.kohsuke.github.GitHub</em><br>
<em>Rate limit now: GHRateLimit{remaining=2042, limit=5000, resetDate=Tue Mar 31 18:31:19 AEDT 2020}</em><br>
<em>Mar 31, 2020 5:59:17 PM FINE org.kohsuke.github.GitHub</em><br>
<em>Rate limit now: GHRateLimit{remaining=2041, limit=5000, resetDate=Tue Mar 31 18:31:19 AEDT 2020}</em><br>
<em>Mar 31, 2020 6:03:04 PM FINE org.kohsuke.github.GitHub</em><br>
<em>Rate limit now: GHRateLimit{remaining=1627, limit=5000, resetDate=Tue Mar 31 18:31:19 AEDT 2020}</em></p>
<p>Or maybe how to check which api calls were made by jenkins to github?</p>
<p>Cheers,<br>
Tung</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/tungjb">@tungjb</a>,</p>
<p>Thanks for being here! Yes, for enterprise instances GitHub application logs can be found in <code>/var/log/github/</code>. Otherwise, you can contact us If you haven’t already, at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a> with the info so requesting this info.</p>

**Comment by tungjb** [comment]
<p>Thanks <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a>  I have tried to contact Git support ticket <span class="hashtag">#611285</span></p>
